code/game/machinery/integrated_tiny_fan.dm

```diff
@@ -0,0 +1,32 @@
+/obj/machinery/integrated_airlock_tiny_fan
+	name = "Integrated Air-based Atmospheric Seal"
+	desc = "A tiny fan linked to the control circuity of the airlock."
+	icon = 'icons/obj/lavaland/survival_pod.dmi'
+	icon_state = "fan_tiny"
+	CanAtmosPass = ATMOS_PASS_YES
+	power_channel = AREA_USAGE_ENVIRON
+	use_power = IDLE_POWER_USE
+	idle_power_usage = 20
+	/// Reference to our parent airlock
+	var/obj/machinery/door/airlock/parent_airlock
```

Comment:
Is this var actually used at all? And if it is needed, you could probably pass it as an argument to Initialize() instead of using locate

Reply:
It was used in an earlier version that I scraped because it was needlessly excessive. I just never actually got rid of it